project/build.properties

```diff
@@ -1 +1 @@
-sbt.version=1.2.8
+sbt.version=1.3.8
```

Comment:
Want to bump `1.3.8` in `lagom/lagom-java.g8` too?

Reply:
No, feel free.

Reply:
https://github.com/lagom/lagom-java.g8/pull/77